docs/source/app_developers_guide/intro_xo_transaction_family.rst

```diff
@@ -2,191 +2,166 @@
 
-URL
----
+* Verify that you can connect to the REST API.
 
-The ``xo`` commands all take a ``--url`` argument that defaults to
-``http://127.0.0.1:8008``. This should be set to the URL of the REST
-API. The *XO* client will send requests to update and query the
-blockchain to this URL.
+  * Docker: See :ref:`confirming-connectivity-docker-label`
 
+  * Ubuntu: See :doc:`/app_developers_guide/installing_sawtooth`
 
-.. seealso::
-  :doc:`/cli/xo`
+  * AWS: See :ref:`confirming-connectivity-aws-label`
 
-Playing XO with the XO Client (Example)
-=======================================
+  .. Important::
 
-Now that we have gone through the basics of the *XO* transaction processor, we are ready
-to play a game. The steps below show you how to set up and play a game using the *XO* CLI.
+     The ``xo`` client sends requests to update and query the blockchain to the
+     URL of the REST API (by default, ``http://127.0.0.1:8008``).
 
-Start the Necessary Components
-------------------------------
+     If the REST API's URL is not ``http://127.0.0.1:8008``, you must add the
+     ``--url`` argument to each ``xo`` command in this procedure.
 
-To play *XO*, ensure that the following components are running and connected:
 
-#. At least one validator
-#. An *XO* family transaction processor
-#. The REST API
-
-Create Players
---------------
+Step 1. Create Players
+----------------------
 
 Create keys for two players to play the game:
 
 .. code-block:: console
 
     $ sawtooth keygen jack
-    $ sawtooth keygen jill
-
-
-The command produces output similar to the following for both players:
-
-.. code-block:: console
-
     writing file: /home/ubuntu/.sawtooth/keys/jack.priv
     writing file: /home/ubuntu/.sawtooth/keys/jack.addr
+
+    $ sawtooth keygen jill
     writing file: /home/ubuntu/.sawtooth/keys/jill.priv
     writing file: /home/ubuntu/.sawtooth/keys/jill.addr
 
 
-Create a Game
--------------
+.. note::
+
+   The output may differ slightly from this example.
+
 
-Create a game with the following command:
+Step 2. Create a Game
+---------------------
+
+Create a game named ``game`` with the following command:
 
 .. code-block:: console
 
     $ xo create game --username jack
 
-To see list of the created games, enter the following command:
+To specify a non-default URL for the REST API, add the ``url`` argument
+to this and all following ``xo`` commands.  This example shows the URL for
+the Docker application environment, ``http://rest-api:8008``.
 
 .. code-block:: console
 
-    $ xo list
+   $ xo create game --username jack --url http://rest-api:8008
+
+.. note::
+
+   The ``--username`` argument is required for ``xo create`` and ``xo take``
+   so that a single user (you) can play as two players. By default,
+   ``<username>`` is the player's user name.
 
-The command outputs a list of the games that have been created:
+   An optional ``--key-dir`` argument can be used to specify a non-default
```

Comment:
I don't know the answer to this question. This information was in the previous version of the doc, so I didn't want to delete it. In my opinion, it doesn't need to be here because this info is in the CLI reference, but perhaps there was a reason someone put this info in this section.

So, what should I do?  Keep it here?  Move it?  Delete it? Or ... ???

Reply:
Thanks, @askmish !